CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
 
 ### Fixed
 * <How do the end-user experience this issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
-* None.
+* Signal that dictionaries have been cleared in the same way List and Set work ([#6205](https://github.com/realm/realm-core/issues/6205), since v.10.4.0)
```

Comment:
How could an end user experience this bug?

Reply:
The final user can only experience this bug if it tries to clear the dictionary and then awaits for the proper flag to be set. But it is up to what the SDKs are doing with this flag, it depends on if it gets exposed, or they just notify via some event. However, the correct behaviour is that every time you clear a collection, that particular flag must be set. For Dictionaries, this is not the case right now.